lightly/utils/benchmarking/knn_classifier.py

```diff
@@ -0,0 +1,99 @@
+from typing import Tuple, Union
+
+import torch
+import torch.nn.functional as F
+from pytorch_lightning import LightningModule
+from torch import Tensor
+from torch.nn import Module
+
+from lightly.models.utils import activate_requires_grad, deactivate_requires_grad
+from lightly.utils.benchmarking.knn import knn_predict
+from lightly.utils.benchmarking.topk import mean_topk_accuracy
+
+
+class KNNClassifier(LightningModule):
```

Comment:
Do we follow any reference implementation here or is this our own contribution?

Reply:
Our own :)